mlrun/runtimes/databricks_job/databricks_wrapper.py

```diff
@@ -40,23 +43,53 @@ def get_task(databricks_run: Run) -> RunTask:
     return databricks_run.tasks[0]
 
 
+def save_credentials(
+    workspace: WorkspaceClient,
+    waiter,
+    host: str,
+    token: str,
+    cluster_id=None,
+    is_finished=False,
```

Comment:
I don't see any benefit in the default values.
```suggestion
    cluster_id: typing.Optional[str],
    is_finished: bool,
```